instrumentation/jdbc/library/src/test/java/io/opentelemetry/instrumentation/jdbc/internal/JdbcConnectionUrlParserTest.java

```diff
@@ -527,6 +559,14 @@ public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
               .setPort(666)
               .setName("orclsn")
               .build(),
+          arg("jdbc:oracle:thin:@(description = (connect_timeout=90)(retry_count=20)(retry_delay=3) (transport_connect_timeout=3000) (address_list = (load_balance = on) (failover = on) (address = (protocol = tcp)(host = orcl.host1)(port = 1521)) (address = (protocol = tcp)(host = orcl.host2)(port = 1521)) (address = (protocol = tcp)(host = orcl.host3)(port = 1521)) (address = (protocol = tcp)(host = orcl.host4)(port = 1521)) ) (connect_data = (server = dedicated) (service_name = orclsn)))")
+              .setShortUrl("oracle:thin://orcl.host1:1521")
+              .setSystem("oracle")
+              .setSubtype("thin")
+              .setHost("orcl.host1")
+              .setPort(1521)
+              .setName("orclsn")
+              .build(),
```

Comment:
this is an attempt to reproduce the example provided in [the issue](https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/12400), but as far as I can tell it seems to work as expected